server/app/repository/AccountRepository.java

```diff
@@ -331,4 +333,19 @@ public int deleteUnusedGuestAccounts(int minAgeInDays) {
 
     return database.sqlUpdate(sql).execute();
   }
+
+  /**
+   * For use in {@link FixApplicantDobDataPathJob}. This will return applicants who have the
+   * incorrect path for applicant_date_of_birth where applicant_date_of_birth points directly to a
+   * number. eg. {applicant:{applicant_date_of_birth: 1038787200000}}
+   */
```

Comment:
a level of nesting is missing in the comment { "applicant_date_of_birth" : {"date" : 100xxx}

Reply:
This is demonstrating the incorrect data case. Where the nested "date" is missing.